Documentation/PageTsconfig/TceForm.rst

```diff
@@ -63,1113 +65,1206 @@ dots ('.'), these must be escaped with backslash.
-   will not be displayed in the content elements form. The
-   content element can still be moved to another column which internally also
-   sets the field :sql:`colPos`. Fields with the TSconfig property
-   :tsconfig:`TCEFORM.<table>.<field>.disable` therefore show the same
-   behaviour as fields of the TCA type :ref:`passthrough <t3tca:columns-passthrough>`.
+    If set, the field is not displayed in the backend form of the record.
+    However, the field can still be set by other means. For example if
+    this property is set:
+    :typoscript:`TCEFORM.tt_content.colPos.disabled = 1` the :guilabel:`Column` field
+    will not be displayed in the content elements form. The
+    content element can still be moved to another column which internally also
+    sets the field :sql:`colPos`. Fields with the TSconfig property
+    :tsconfig:`TCEFORM.<table>.<field>.disable` therefore show the same
+    behaviour as fields of the TCA type :ref:`passthrough <t3tca:columns-passthrough>`.
 
-   table level, example:
-      `TCEFORM.tt_content.header.disabled`
+    table level, example:
+        `TCEFORM.tt_content.header.disabled`
 
-   table and record type level, example:
-      `TCEFORM.tt_content.header.types.textpic.disabled`
+    table and record type level, example:
+        `TCEFORM.tt_content.header.types.textpic.disabled`
 
-   Flex form sheet level. If set, the entire tab is not rendered, example:
-      `TCEFORM.tt_content.pi_flexform.myext_pi1.sDEF.disabled`
+    Flex form sheet level. If set, the entire tab is not rendered, example:
+        `TCEFORM.tt_content.pi_flexform.myext_pi1.sDEF.disabled`
 
-      Where `sDEF` is the sheet name. For a description
-      see the section :ref:`tceformApplyPropertiesFlexForm` on this page.
+        Where `sDEF` is the sheet name. For a description
+        see the section :ref:`tceformApplyPropertiesFlexForm` on this page.
+
+    Flex form field level, example:
+        `TCEFORM.tt_content.pi_flexform.myext_pi1.sDEF.settings\.myfield.disabled`
 
-   Flex form field level, example:
-      `TCEFORM.tt_content.pi_flexform.myext_pi1.sDEF.settings\.myfield.disabled`
+        Where `sDEF` is the sheet name. For a description
+        see the section :ref:`tceformApplyPropertiesFlexForm` on this page.
 
-      Where `sDEF` is the sheet name. For a description
-      see the section :ref:`tceformApplyPropertiesFlexForm` on this page.
+..  _tceform-disabled-example:
 
-:aspect:`Example`
-   .. code-block:: typoscript
-      :caption: EXT:site_package/Configuration/page.tsconfig
+Example: Disable editing of the page title
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-      TCEFORM.pages.title {
-         # The title field of the pages table is not editable
-         disabled = 1
-      }
+..  code-block:: typoscript
+    :caption: EXT:site_package/Configuration/page.tsconfig
 
+    TCEFORM.pages.title {
+        # The title field of the pages table is not editable
```

Comment:
Just a note, no action needed: When disabling a field, it is not visible anymore. "Not editable" reads as "readonly".